modin/experimental/engines/omnisci_on_native/io.py

```diff
@@ -12,10 +12,10 @@
 # governing permissions and limitations under the License.
 
 """
-Module houses ``OmnisciOnRayIO`` class.
+Module houses ``OmnisciOnNativeIO`` class.
 
-``OmnisciOnRayIO`` is used for storing IO functions implementations with OmniSci backend
-and Ray engine.
+``OmnisciOnNativeIO`` is used for storing IO functions implementations with Arrow backend
```

Comment:
Is it OmniSci backend and Native engine now?

Reply:
yes, after #2957 is merged OmniSci will be storage_format.

Reply:
fixed